src/pages/index.tsx

```diff
@@ -1,118 +1,23 @@
-import Image from "next/image";
-import { Inter } from "next/font/google";
+import type { GetStaticProps, InferGetStaticPropsType } from 'next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import type { Locale } from '@/lib/seo'
+import { localesSeoData } from '@/lib/seo'
+import Home from '@/components/pages/Home'
 
-const inter = Inter({ subsets: ["latin"] });
+export const getStaticProps: GetStaticProps = async ({ locale }) => {
+  const seoData = localesSeoData[(locale as Locale) ?? 'en']
 
-export default function Home() {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, ['common', 'about_us'])),
```

Comment:
```suggestion
      ...(await serverSideTranslations(locale, ['common', 'about_us'])),
```
non-null assertionぬけるとおもいます！